flask_iiif/restful.py

```diff
@@ -146,7 +147,7 @@ def get(self, version, uuid, region, size, rotation, quality, image_format):
         IIIFImageAPIWrapper.validate_api(**api_parameters)
 
         # build the image key
-        key = "iiif:{0}/{1}/{2}/{3}/{4}.{5}".format(
+        key = u"iiif:{0}/{1}/{2}/{3}/{4}.{5}".format(
```

Comment:
here is present an old error in the order of this parameter. The IIIF standard expect this order: 
```
{scheme}://{server}{/prefix}/{identifier}/{region}/{size}/{rotation}/{quality}.{format}
```
As you can see rotation and quality are inverted. i don't know if we can fix now.